### PR TITLE
caf: click_detector: configurable click intervals

### PR DESCRIPTION
**Pull request by Embla-Flatlandsmo**
Signed-off-by: Embla Flatlandsmo <embla.flatlandsmo@nordicsemi.no>

### CODE REVIEW

File: subsys/caf/modules/Kconfig.click_detector [hunk 1]
```diff
@@ -17,6 +17,14 @@ config CAF_CLICK_DETECTOR_DEF_PATH
 	help
 	  Location of configuration file for click detector module.
 
+config CAF_CLICK_DETECTOR_LONG_CLICK_MSEC
+	int "Minimum button hold time to decide that the click is long"
+	default 3000
```
Comment (pdunaj) [inline subsys/caf/modules/Kconfig.click_detector:22]: default 5000

Let's keep the value as it was to ensure nothing breaks

**Review by pdunaj — CHANGES_REQUESTED** [review]
Please change the commit title  and message to:
```
caf: click_detector: configurable click intervals

Allow configuration to change the click intervals.

Signed-off....
```

**Comment by pdunaj** [comment]
Thanks for the contribution @Embla-Flatlandsmo :)
This is something we should have done from the beginning.

File: subsys/caf/modules/Kconfig.click_detector (hunk 1 above)
Comment (Embla-Flatlandsmo) [reply subsys/caf/modules/Kconfig.click_detector]: Good thinking! I wanted to make it 3000 as 5000 feels too long in my project, but then again, there are probably other applications where 5000 is good :) It's fixed now_

**Review by pdunaj — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Comment by pdunaj** [comment]
Hi @Embla-Flatlandsmo , could you rebase the PR? It would be good to get it merged.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**